Cargo.toml

```diff
@@ -1,7 +1,20 @@
 [package]
 name = "oci-spec"
-version = "0.1.0"
+version = "0.3.0"
 edition = "2018"
+authors = [
+    "Furisto",
```

Comment:
@Furisto do you have additional metadata for here to add?

Reply:
No, that's it.